src/vs/workbench/contrib/chat/common/chatModel.ts

```diff
@@ -1360,11 +1362,16 @@ export class ChatModel extends Disposable implements IChatModel {
 
 	// TODO to be clear, this is not the same as the id from the session object, which belongs to the provider.
```

Comment:
ugh I think this TODO is very obsolete, no async init today, adding to debt list...